models/agol.js

```diff
@@ -254,6 +266,46 @@ var AGOL = function (koop) {
   }
 
   /**
+   * Wraps Cache.updateInfo for easier testing
+   * @param {string} key - a table name in the db
+   * @param {object} info - the info to store in the table
+   * @param {function} callback  - calls back with err or nothing
+   */
+  agol.updateInfo = function (key, info, callback) {
+    koop.Cache.getInfo(key, info, callback)
```

Comment:
Thanks for spotting that. That's laziness in copy/pasting on the docs. Will correct.